src/billing/List/ProjectListItem.tsx

```diff
@@ -0,0 +1,131 @@
+import { Clickable, Icon } from '@terra-ui-packages/components';
+import _ from 'lodash/fp';
+import * as qs from 'qs';
+import React, { useState } from 'react';
+import { BillingProjectActions, BillingProjectActionsProps } from 'src/billing/List/BillingProjectActions';
+import { billingRoles, isCreating, isDeleting, isErrored } from 'src/billing/utils';
+import { BillingProject } from 'src/billing-core/models';
+import { CloudProviderIcon } from 'src/components/CloudProviderIcon';
+import { InfoBox } from 'src/components/InfoBox';
+import { Ajax } from 'src/libs/ajax';
+import colors from 'src/libs/colors';
+import Events, { extractBillingDetails } from 'src/libs/events';
+import * as Nav from 'src/libs/nav';
+import * as Style from 'src/libs/style';
+import { isKnownCloudProvider } from 'src/workspaces/utils';
+
+const listItemStyle = (selected, hovered) => {
+  const style = {
+    ...Style.navList.itemContainer(selected),
+    ...Style.navList.item(selected),
+    ...(selected ? { backgroundColor: colors.dark(0.1) } : {}),
+    paddingLeft: '2rem',
+  };
+  if (hovered) {
+    return {
+      ...style,
+      ...Style.navList.itemHover(selected),
+    };
+  }
+  return style;
+};
+
+export interface ProjectListItemProps {
+  project: BillingProject;
+  isActive: boolean;
+  billingProjectActionsProps: BillingProjectActionsProps;
+}
+
+export const ProjectListItem = (props: ProjectListItemProps) => {
```

Comment:
Individual project names in the left-hand list:

![image](https://github.com/user-attachments/assets/92e592b3-32a9-4446-85a9-10e06ebf0cbb)